code/src/utest/plugin/lsttokens/sizeadjustment/AbbTokenTest.java

```diff
@@ -75,15 +75,15 @@ public void testInvalidInputEmpty() throws PersistenceLayerException
 	public void testValidInputs() throws PersistenceLayerException
 	{
 		assertTrue(parse("Nieder�sterreich"));
-		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB));
+		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB_KR));

```

Comment:
It might have predated the git move. Quite likely it was a file charset change that did the damage. We can deal with it in a separate PR though.